Gemfile

```diff
@@ -24,6 +24,7 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'orderly'
```

Comment:
Might be handy to have for later tests too

Reply:
I've heard of this! Excited to try it out.